### ISSUE TITLE
Naming inconsistent for Twitter Creds

### ISSUE THREAD

**Issue by rhuss** [syndesisio/syndesis-ui#1039]
![image](https://user-images.githubusercontent.com/99080/31057238-cccffe68-a6df-11e7-89a4-06007484637b.png)

**Comment by dongniwang** [comment]
@rhuss 
Are you suggesting they should be "Consumer Key" and "Consumer Secret"? If yes, I agree we should change it so that it's easier to recognize from a user point of view.

**Comment by gashcrumb** [comment]
The limitation here is that the form is a generic thing client-side.  We could introduce a client-side mapping of target system -> appropriate form naming, but maybe it'd be better if this form config lived in the connector as well and then driven by the backend instead.

**Comment by rhuss** [comment]
@gashcrumb I agree, this should come out of the connector. Do we already have an interaction with the backend for this page ? I mean, do you already get the list of OAuth setting sections from the backend or is it hardcoded in the client ?

If not yet, we might add a field 'usesOAuth' to the connector and then provide the labels for the key and secret. This then could be extracted by the backend and delivered to the UI via an own endpoint.

**Comment by gashcrumb** [comment]
The UI currently just fetches an array of settings from `/v1/setup/oauth-apps` which unfortunately is defined like [this](https://github.com/syndesisio/syndesis-ui/blob/master/src/app/model.ts#L298-L304), I think we'd want to change this to have a separate `properties` and configuration attribute so it's more consistent with our other objects and gives us an attribute that'd contain the form configuration for each OAuth endpoint.  I think not with a single flag, I'd suggest either specific endpoints (like `/v1/setup/security/<type>`) or an enum value.
